projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -102,6 +102,84 @@ class DashBackups extends Component {
 		} );
 	};
 
+	trackRedeemCouponButtonView = () => {
+		const { partnerCoupon } = this.props;
+
+		analytics.tracks.recordEvent( 'jetpack_wpa_aag_redeem_coupon_button_view', {
```

Comment:
Kind of a half-hearted suggestion: What about changing this to be partner coupon specific so we don't conflict with generic coupons in the future? I know we've started working with seasonal coupons on Jetpack.com. Perhaps we want to highlight them in the plugin at some point?
E.g.: "Claim your Black Friday offer"

`jetpack_wpa_aag_redeem_coupon_button_view` => `jetpack_wpa_aag_redeem_partner_coupon_button_view`

Reply:
Done.